src/scss/_learn.scss

```diff
@@ -0,0 +1,3 @@
+.footnotes {
+    font-size: 0.75rem;
```

Comment:
i dont totally know whats going on but can we not use tailwind for this?
`text-sm` or maybe xxs?

Reply:
I tried. Unfortunately, the markdown processor renders footnotes after all the classes are applied. The only way I can affect the footnotes is to change the behavior of the default class it applies like this. I've tried stuff like adding `markdown=1` to the HTML, wrapping it in other tags to apply the other classes... Goldmark doesn't have a class-apply function for non-headings, from what I've been able to find.